src/ceph-detect-init/ceph_detect_init/docker/__init__.py

```diff
@@ -0,0 +1,3 @@
+def choose_init():
+    """Returns 'docker' because we are running in a container"""
+    return 'docker'
```

Comment:
I believe it's clearer to return "none" since there is no init system within a container.
Also, people might use other container engine solution and might get different results.
Otherwise the rest looks good me! Thanks!